packages/core/contracts/insured-bridge/non-bridge-helpers/BridgePoolSynchronizer.sol

```diff
@@ -0,0 +1,32 @@
+pragma solidity ^0.8.0;
+
+interface BridgePoolLike {
+    function syncUmaEcosystemParams() external;
+
+    function syncWithBridgeAdminParams() external;
+}
+
+/**
+ * @notice Small helper contract to make calling sync methods on a set of bridge pools easier. When one pool needs to be
+ * sync it is likely that all ecosystem pools also need to be synced. This contract lets the caller do this in a batch.
+ */
+contract BridgePoolSynchronizer {
```

Comment:
Seems like overkill to deploy a full contract just to send transactions like this. Why not use Multicall2 since we already have a pretty deep integration with that in our bots and in the SDK?

Reply:
indeed; multicall is the right way to do this or using DSProxy. in any case though you still need to "implementation" that you will be calling. I'll add a script to this PR that lets you do this.

Reply:
in fact using multicall means we dont actually need the solidity file. this is simply a script now :)

Reply:
💯 Yeah, just didn't want to deploy a single-use contract!